AUTHORS.rst

```diff
@@ -11,6 +11,7 @@ CumulusCI is developed by the Salesforce.org Release Engineering team:
 * Christian Carter <christian.carter@salesforce.com>
 * David Glick <dglick@salesforce.com>
 * James Estevez <jestevez@salesforce.com>
+* Josh Kofsky <jkofsky@salesforce.com>
```

Comment:
@jofsky heh, I forgot about this file. Could you add the rest of the team too please? (ask them to make sure it's okay to use their email addresses)